### DISCUSSION TITLE
BelongsTo or Select with searchable() does not work in action modal

### DISCUSSION THREAD

**Discussion by jonnigg** [laravel/nova-issues#6400] [General]
When using BelongsTo::make()->searchable() or Select::make()->searchable() in the fields function of an action the searchable input box is will not become focusable (won't allow text input). As a result. The search functionality does not work. The searchable() functionality does work in normal forms however. It seems that this issue is limited to being used in the modal that is presented when running actions. I have attached a few screenshots below. 

FWIW - It almost feels like a z-index issue such that the input is behind some other element.

Any help is greatly appreciated.

I am using the following:
Nova: v4.33.3 (Silver Surfer)

Screenshots:
![image](https://github.com/laravel/nova-issues/assets/3459294/34cb8a1b-21f6-42c9-94b0-43169a813855)

![image](https://github.com/laravel/nova-issues/assets/3459294/3d7e05cc-f4c2-4cf7-a344-49d1e69a00d0)

**Comment by crynobone** [comment]
Resolved via https://github.com/laravel/nova-issues/issues/6401#issuecomment-2127654000
